metaflow/main_cli.py

```diff
@@ -349,83 +348,79 @@ def sandbox(profile):
+        use_s3 = click.confirm('\nConfigure Amazon '
+                                'S3 as default datastore (required for AWS Batch)?',
                               default=True, abort=False)
         if use_s3:
-            echo('\tAWS S3', fg='cyan')
-            datastore_s3_root =\
-                click.prompt('\tPlease enter the bucket prefix to use for your '
-                             'flows')
-
-            datatools_s3_root =\
-                click.prompt('\tPlease enter the bucket prefix to use for your '
-                             'data',
-                             default='%s/data' % datastore_s3_root)
-
             env_dict['METAFLOW_DEFAULT_DATASTORE'] = 's3'
-            env_dict['METAFLOW_DATASTORE_SYSROOT_S3'] = datastore_s3_root
-            env_dict['METAFLOW_DATATOOLS_SYSROOT_S3'] = datatools_s3_root
-
-            # Batch configuration (only if S3 is being used).
+            env_dict['METAFLOW_DATASTORE_SYSROOT_S3'] =\
+                click.prompt('\t' + cyan('METAFLOW_DATASTORE_SYSROOT_S3:') + 
+                  ' Amazon S3 url for metaflow datastore (s3://<bucket>/<prefix>)')
+
+            env_dict['METAFLOW_DATATOOLS_SYSROOT_S3'] =\
+                click.prompt('\t' + cyan('METAFLOW_DATATOOLS_SYSROOT_S3:') + 
+                  yellow(' (optional)') + ' Amazon S3 url for metaflow datatools (s3://<bucket>/<prefix>)',
+                    default='%s/data' % env_dict['METAFLOW_DATASTORE_SYSROOT_S3'],
+                    show_default=True)
+            # AWS Batch configuration (only if Amazon S3 is being used).
             use_batch =\
-                click.confirm('\nDo you want to use AWS Batch for compute?',
+                click.confirm('\nConfigure AWS Batch for compute?',
                               default=True, abort=False)
             if use_batch:
-                echo('\n\tAWS Batch', fg='cyan')
-                job_queue = click.prompt('\tPlease enter the job queue to use '
-                                         'for batch')
-                default_image =\
-                    click.prompt('\tPlease enter the default container image '
-                                 'to use')
-                container_registry =\
-                    click.prompt('\tPlease enter the container registry')
-                ecs_s3_role =\
-                    click.prompt('\tPlease enter the IAM role to use for the '
-                                 'container to get AWS S3 access')
-
-                env_dict['METAFLOW_BATCH_JOB_QUEUE'] = job_queue
-                env_dict['METAFLOW_BATCH_CONTAINER_IMAGE'] = default_image
-                env_dict['METAFLOW_BATCH_CONTAINER_REGISTRY'] =\
-                    container_registry
-                env_dict['METAFLOW_ECS_S3_ACCESS_IAM_ROLE'] = ecs_s3_role
-
+                env_dict['METAFLOW_BATCH_JOB_QUEUE'] =\
+                    click.prompt('\t' + cyan('METAFLOW_BATCH_JOB_QUEUE:') + 
+                      ' AWS Batch Job Queue')
```

Comment:
Could we be a bit more specific here? I thought there was some confusion what entry to have here (the arn or the queue name or what).

Reply:
Both of them work equally well. For any of the configs, the user can specify ARNs or names as they are interchangeable.